Source/Initialization/WarpXAMReXInit.cpp

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2020 Axel Huebl
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#include "Initialization/WarpXAMReXInit.H"
+
+#include <AMReX_ParmParse.H>
+
+
+namespace {
+    /** Overwrite defaults in AMReX Inputs
+     *
+     * This overwrites defaults in amrex::ParamParse for inputs.
+     */
+    void
+    overwrite_amrex_parser_defaults()
+    {
+        amrex::ParmParse pp("amrex");
+
+        // https://amrex-codes.github.io/amrex/docs_html/GPU.html#inputs-parameters
+        bool abort_on_out_of_gpu_memory = true; // default: false
```

Comment:
I got confused with this:
```suggestion
        bool abort_on_out_of_gpu_memory = true; // default: true
```

Reply:
I mean:
```suggestion
        bool abort_on_out_of_gpu_memory = true; // AMReX default: false
```

Reply:
Ah ok ok, then it makes sense! Thanks